classes/app.rb

```diff
@@ -0,0 +1,146 @@
+require_relative 'student'
+require_relative 'teacher'
+require_relative 'rental'
+class App
+  attr_reader :books, :persons, :rentals
+
+  def initialize
+    @books = []
+    @persons = []
+    @rentals = []
+  end
+  puts 'Select an option'
+
+  def list_people
+    if @persons.empty?
+      puts 'no person added '
+    else
+      @persons.each { |person| puts("name: #{person.name}") }
+    end
+  end
+
+  def list_books
+    if @books.empty?
+      puts 'book list is empty'
+    else
+      @books.each { |book| puts "Title: #{book.title} Author: #{book.author}" }
+    end
+  end
+
+  def create_person
+    puts "Create a Teacher or a Student
+        [1] -> Student
```

Comment:
[OPTIONAL] Rise an error if you give non integer value for Student/Teacher age. As you can here in these images:

TEACHER

> ![Screen Shot 2022-08-27 at 05 45 02](https://user-images.githubusercontent.com/69011963/187007824-740789e9-ac44-4345-90c5-663856d625c1.png)
> 
STUDENT
> ![Screen Shot 2022-08-27 at 05 44 30](https://user-images.githubusercontent.com/69011963/187007835-df1f9d66-6b03-48a0-846e-d6e271fb5d13.png)
> 
> 
>

Reply:
Hey, I saw your mail! It's an optional requirement you can leave it if you want. But if you want to apply it I would suggest you look for this [link](https://www.geeksforgeeks.org/raising-exceptions-in-ruby/).